### PR TITLE
BY-COVID Pathways workflow with MINERVA

### PR DESCRIPTION
**Pull request by hexylena**

### CODE REVIEW

File: topics/transcriptomics/tutorials/minerva-pathways/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+>    > If that still doesn't work, please check that the Galaxy server you are using is updated to 24.0 or later.
+>    >
+>    {: .tip}
+>
+{: .hands_on}
+
+### Analysis
+
+
+TODO
```
Comment (hexylena) [inline topics/transcriptomics/tutorials/minerva-pathways/tutorial.md:327]: @mjostaszewski @MattiHoch  if y'all can write something on how to do the data interpretation with MINERVA that would be great (which buttons to click, what do you suggest to look at (e.g. a pathway or component or two that the user should open and examine some data in the left hand menu or so)

**Comment by hexylena** [comment]
@bgruening would you like to merge it since you're also in by-covid? :)

**Comment by bgruening** [comment]
Ok, will read it and take care of ittoday, is that ok?

**Review by bgruening — APPROVED** [review]
Very cool!

**Comment by hexylena** [comment]
Thanks for the corrections @bgruening ! someday I will install a spell checker.

**Pull request merged**